src/darwin/Framework/CHIP/templates/partials/CHIPCallbackBridge.zapt

```diff
@@ -0,0 +1,49 @@
+class CHIP{{name}}CallbackBridge : public Callback::Callback<{{name}}Callback> {
+public:
+    CHIP{{name}}CallbackBridge(ResponseHandler handler, dispatch_queue_t queue, bool keepAlive = false)
+        : Callback::Callback<{{name}}Callback>(CallbackFn, this)
+        , mHandler(handler)
+        , mQueue(queue)
+        , mKeepAlive(keepAlive)
+        {}
+
+    ~CHIP{{name}}CallbackBridge() {};
+
+    static void CallbackFn(void * context{{#if type}}, {{#if (isStrEqual type "Boolean")}}bool{{else}}{{asUnderlyingZclType type}}{{/if}} value{{/if}})
```

Comment:
For the record, the `{{#if (isStrEqual type "Boolean")}}` macro can be removed once #8818 lands